### ISSUE TITLE
Installing new VSIX should be blocked while waiting for another VSIX install.

### ISSUE THREAD

**Issue by dlebu** [microsoft/vscode#40326]
VSCode Version: Insiders 1.19.0
OS Version: MacOS

Steps to Reproduce:

Install a large extension from vsix (> 100mb such that it takes a long time).
Before the confirmation message, try to install it again.

Expected: User should be blocked on waiting, before he is able to install the second time.
Actual: Same VSIX can be installed again, corrupting the installation.

@ramya-rao-a

**Comment by dlebu** [comment]
FYI @k7shanmugam

**Comment by dlebu** [comment]
Related #40257

**Comment by VSCodeTriageBot** [comment]
We closed this issue because we [don't plan to address it](https://aka.ms/vscode-out-of-scope) in the foreseeable future. If you disagree and feel that this issue is crucial: we are happy to listen and to reconsider.

If you wonder what we are up to, please see our [roadmap](https://aka.ms/vscoderoadmap) and [issue reporting guidelines](https://aka.ms/vscodeissuereporting).

Thanks for your understanding, and happy coding!
